clcache.py

```diff
@@ -186,7 +186,7 @@ def acquire(self):
             elif time.time() - startTime > self._timeoutMs/1000:
                 raise ObjectCacheLockException("Timeout waiting for file lock")
             else:
-                pollDelay = random.uniform(0.01, 0.1)
+                pollDelay = random.uniform(0.1, 1.0)
```

Comment:
I'm curious: what does this fix? SMB connections don't like polling too often for some reason?